config.json.template

```diff
@@ -6,6 +6,8 @@
         "log_level": "warn",
 
         "authentication": {
+            "eviction_check_interval_in_seconds": 60,
```

Comment:
this is not in the default documentation

Reply:
Yep. Will add.

I also need to remove this template file since it's part of the server now.

Reply:
even better.

Reply:
done